### DISCUSSION TITLE
Development Request

### DISCUSSION THREAD

**Discussion by jbcurler2010** [Niek/obs-web#82] [General]
Hello! Love the OBS Web tool!

Wondering how difficult it would be to put a custom page together?

We have a large OBS Portable deployment with multiple OBS Portable instances at multiple sites.

Looking to present our customers with a simple START/STOP streaming page using websockets.

Ideally a config page for up to 8 OBS instances (number of OBS instances, OBS Instance Name, IP,  & Port) and a single control page to be able to START/STOP ALL or START/STOP individual instances. And maybe a custom logo.

Thanks in advance for any input you may have.

**Comment by Niek** [comment]
Hi, this is currently not possible in the UI, but you could build something on top of OBS-web or by building your own UI on top of https://github.com/obsproject/obs-websocket
